Gemfile.lock

```diff
@@ -449,6 +449,7 @@ GEM
     naught (1.1.0)
     net-http-persistent (2.9.4)
     newrelic_rpm (5.0.0.342)
+    nio4r (2.3.1)
```

Comment:
What prompted this gem addition?

Reply:
I keep taking it out it keeps sneaking back in. hang on.

Reply:
We discussed this is actually needed for actioncable.